kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -278,7 +278,7 @@ export default function useQuizCreation() {
    */
   function saveQuiz() {
     const totalQuestions = get(allSections).reduce((acc, section) => {
-      acc += parseInt(section.question_count);
```

Comment:
@nucleogenesis - this value again feels like something we maybe shouldn't even store in the backend, as all it seems to represent is a user intention of how many questions there _should_ be in the section?

Reply:
Yeah that's a good point - it can always be derived from the length of questions anyway.

@AlexVelezLl I'll include the removal of saving this in my PR #12200